client/src/styles/Header/Header.scss

```diff
@@ -0,0 +1,163 @@
+nav {
+  margin: 0;
+  overflow: hidden;
+  background: black;
+  height: 50px;
+}
+
+.nav-content {
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 0 15px;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 2px;
+
+  a {
+    display: block;
+    font-size: 14px;
+    line-height: 44px;
+    text-decoration: none;
+    transition: all 0.3s;
+    color: #ffffff;
+  }
+}
+
+.icon {
+  opacity: 0.8;
+  margin-top: 8px;
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+.nav-links {
+  width: 75%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  a {
+    opacity: 0.8;
+
+    &:hover {
+      opacity: 1;
+    }
+  }
+}
+
+.hra {
+  display: none;
+}
+
+.nav-icon {
+  display: none;
+  grid-gap: 5px;
+  grid-template-columns: auto;
+  margin-top: 2px;
+  padding: 17px 0;
+  height: 10px;
+  cursor: pointer;
+  transition: all 0.3s;
+}
+
+.bar {
+  height: 1px;
+  width: 18px;
+  background: white;
+  transition: 0.5s;
+}
+
+.search {
+  cursor: pointer;
+  transition: all 0.3s;
+  opacity: 0.8;
+  margin-top: 11px;
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+.search1 {
+  cursor: pointer;
+  transition: all 0.3s;
+  opacity: 0.8;
+  margin-top: 10px;
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+@media (max-width: 768px) {
+  .nav-content {
+    padding: 0 15px;
+  }
+
+  .nav-links {
+    position: fixed;
+    top: 44px;
+    right: 0;
+    height: 0;
+    width: 100%;
+    background: black;
+    flex-direction: column;
+    justify-content: flex-start;
+    transition: all 2s cubic-bezier(0.19, 1, 0.22, 1);
+  }
+
+  .show .nav-links {
+    height: 100%;
+    padding-top: 30px;
+  }
+
+  .nav-links a {
+    height: 0;
+    width: 0;
+    opacity: 0;
+    overflow: hidden;
+    margin-right: 50px;
+    margin-left: 50px;
+    transition: opacity 1.5s, height 2s;
+  }
+
+  .show .nav-links a {
+    opacity: 1;
+    width: auto;
+    height: auto;
+  }
+
+  .hra {
+    color: white;
+  }
+
+  .nav-icon {
+    order: 1;
+    display: grid;
+  }
+
+  .icon {
+    order: 2;
+    opacity: 0.9;
+  }
+
+  .search1 {
+    order: 3;
+  }
+
+  .search {
+    display: none;
+  }
+}
+
+.show {
+  .one {
+    transform: rotate(45deg) translateY(4px);
+  }
+
+  .two {
+    transform: rotate(-45deg) translateY(-4px);
+  }
```

Comment:
What does `.one` and `.two` mean?

Reply:
It's for this cross at the upper left, and the javascript I added in the useEffect is for it too.Upon click the js attaches the show class which adds .one and .two to it.
![105450975-aacd3f80-5ca1-11eb-8f13-c21ea729a3a5__01](https://user-images.githubusercontent.com/48972593/106255557-30bc2e00-6240-11eb-8ac5-1f3fd43db1c9.png)

Reply:
Okay, cool. Better class names might be great. Also, not a good idea to use direct DOM reference. Can we use `useState`? I can help you if you need some code.